bot/utils/voice_recognition.py

```diff
@@ -0,0 +1,101 @@
+import os
+from google.cloud import speech
+import requests
+import logging
+import json
+from tempfile import NamedTemporaryFile
+import subprocess
+
+TOKEN = os.environ["TOKEN"]
+
+LANG = "ru-RU"
+SAMPLE_RATE_HERTZ = 48000
+ENCODING_TYPE = "LINEAR16"
+CHANNEL_COUNT = 1
+ENABLE_AUTOMATIC_PUNCTUATION = True
+
+logger = logging.getLogger(__name__)
+
+speechClient = speech.SpeechClient()
```

Comment:
Variable in function should be [lowercase](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

Reply:
Fixed.